### PR TITLE
EDU-12098 - Creating vtex-io-documentation-making-a-custom-component-available-in-site-editor.mdx

### PR DESCRIPTION
**Pull request by barbara-celi**
#### Types of changes
- [X] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

[EDU-12098](https://vtex-dev.atlassian.net/browse/EDU-12098)

[EDU-12098]: https://vtex-dev.atlassian.net/browse/EDU-12098?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/guides/vtex-io/Storefront-Guides/concepts-1/vtex-io-documentation-making-a-custom-component-editable-in-the-site-editor.mdx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+In this guide, you will learn how to configure the schema definitions in your components, allowing users to customize them directly in the Site Editor.
+
+## Before you begin
+
+Ensure your store has a [Store Theme](https://developers.vtex.com/docs/guides/vtex-io-documentation-store-theme) developed with [VTEX IO](https://developers.vtex.com/docs/guides/vtex-io-documentation-what-is-vtex-io) and [Store Framework](https://developers.vtex.com/docs/guides/store-framework), according to the [Storefront](https://developers.vtex.com/docs/guides/getting-started-3) guide.
+
+Check if the [Builders](https://developers.vtex.com/docs/guides/vtex-io-documentation-builders) are properly installed in your Store Theme. You must have at least the following builders configured:
+
+- [store builder](https://developers.vtex.com/docs/guides/vtex-io-documentation-store-builder): Enables the development of Store Framework storefronts.
+- [react builder](https://developers.vtex.com/docs/guides/vtex-io-documentation-react-builder): Used to develop apps with [React](https://react.dev/) when your project requires customized frontend solutions.
```
Comment (carolinamenezes) [inline docs/guides/vtex-io/Storefront-Guides/concepts-1/vtex-io-documentation-making-a-custom-component-editable-in-the-site-editor.mdx:21]:
```suggestion
- Ensure your store has a [Store Theme](https://developers.vtex.com/docs/guides/vtex-io-documentation-store-theme) developed with [VTEX IO](https://developers.vtex.com/docs/guides/vtex-io-documentation-what-is-vtex-io) and [Store Framework](https://developers.vtex.com/docs/guides/store-framework), according to the [Storefront](https://developers.vtex.com/docs/guides/getting-started-3) guide.
- Check if the [Builders](https://developers.vtex.com/docs/guides/vtex-io-documentation-builders) are properly installed in your Store Theme. You must have at least the following builders configured:
    - [store builder](https://developers.vtex.com/docs/guides/vtex-io-documentation-store-builder): Enables the development of Store Framework storefronts.
    - [react builder](https://developers.vtex.com/docs/guides/vtex-io-documentation-react-builder): Allows developing [React](https://react.dev/) frontend components. 
```

**Review by carolinamenezes — COMMENTED** [review]
Sugestão de usar um único Scrollycoding conforme
[Sugestão 1](https://play.codehike.org/#N4IgtgJgHiBcIFokB0B2AXAlugNgU1gAJkQBZAQwGtNUBzQ8wgYwFcBndAezGe4AdOqPBkJ4I2cgCN8hGoXQALPIQDK2ZQFFxXAE4k0bHC1pESYKjVoJyCVh263+g4egRiJ0vAhoJFXtupu2px6IGgKmBAQwkQAZuQ4bHhoTDp45OhiAILopiAATAAM+QAsCIUAbAgAjADMACr5tbC1+bCFAKwAdAAcHbUAWvqoLHwQGdm5xAXFZZU1DU0tbZ29-UNhQlBMeDp8UyQAMuk6qIQKnADu8pwMODhXvGACQiJ2XGCYAF4ZmIKE4jSTFwAE9ZGc-Kp1IQtNgQl1hkgEGg0PVbgkHtcmE5XuhmOwPt9fv85ABtNSZGHBHQAXQAFAp0Og+GxYAB6NlKHB8LoAN0yUC62LAbOEbPQLF0mASbICmSCcJ0CE4vN2vMweEuSHyAE4dQARSR4SQ6sAAMSYXwAqvUQQA5ACKOqgAAU8NUAPwAMklYAA+mxOCwdDsALzkSWcYV8fCZACUABpCCCg4QhGIbgC8LEaMpIWwmEpzIRYiFRORC6JtFIZHwdJw+Lt0GDyKgIIQ0rFdsIdvIlEksznUNg-qg2OC+8oAAY0TI6eI7NhdABWgdQU5LmHwCNQaAAkhCIuPaCxIngkymWIRLlucIR8ORTudHlxeKgcye0pPCAWi4xokOI6COOciXjoTwvC4bBJhiVyWIQ7C7OOr7vNw3x5kWAKYECoITpCFKaNSO5oAAxCRhAAELZiEyiXoQRq0DQKKoAgMJjsGtFBuB9hfgo5CqgwhDkroyj1EWeD0oyzKshy0Sqg8jY6Eu-J4IKwpshAUZsGyJ5ntpKlQN4nAIJprBgC4xIsTxXh+OZcZZvJDYZjeihCQAavUGgABqEHuADyklMiy7IaXgjmKcpApCtwGlaTpp7RPpApGSZUYsOZGCWQglx8a4mBsAgBlGfZrbtsJNGEGaOjkOZlwhJQgXSSFcl4ApSF8lF6mmdpumJbKIkILE1W1fViYMEw2I6OIdCZpC5WdvWGCNcFslha1TlKR1qnRSK3XxXpOl4EylgIBwj6ZBACC1PZvV4DurEAMJKEwlCyLE36khRp44NESnLTJoXhe1Bk7bFTA9QleBJapKWmelFlYIICCSN9v1sCVX51htOBgjQZ33BmoFcaoImEGJeDmV0hAAJqpmABLnPxygZPe6QcN+pb3HBM0o1uaNvh+HEQLAaCEGLrGktZ9Go7s-3NWtbWbSDXVxbd0OGX8qVmQjo6nQNvM-bscZEBoqA1lD34tQpGV4pw70qKTVU1Zq9U-iJQ2COgS6i+LQlpBWeIG79curUDSudTFe1q2yRWa3DNtZf7wLIzLOjG4QVpJO2r5Ww2DB8Cy17YAoQkAErpMCIdskn6BdHJbL2TlwjJsTWPLngwIdngACOp5pOOqGfF8GYexgwjtoGRiI2OVPMQAPA9AASXQqKknD3CC2LTbQAB8zFkSTeB8IQ1SEI9-tYDNYH4vYPDRs4GDMWaZbX9IUava+FY7PsCFJOBg9EmntBUQUABA6DxJCcuAcIIPzxP3F4ARPAljLJCN+L0qZiXyjA3EP4LgsB+uCJgRhoiCV-BTRgigWawUuOOdwl96CAg7rhOQ+FoSwl0MRVA1QqYPQvszNMmpkGG3AiwpQhApzVwrugDc9ZOB4kYcCEIIIqbPy5tcSEfAMhzjOFOGubJ764m0sAB6OIXB2mdgAXzZDQaIgovZQCnCossqkaoxnPMg8CjADEuDTM7dsU4HoEm4CY54sCpxJkhJzX6bsdAsGBBxWQ45yDjk5piVkPtiCoHyFTPyjZDzThsdtexG4cwyFKoOXMLdgw3w+NgnxyS8JiNSdzegHAj7e13OuKcU50BQHQGgGuGTADEpIQQAvBuACGdupGA2AZLFoQEZEzCCBNviEyCj8zhzM2fM8Etiuj2LQN0qczEkT7wPvqbMNB4KQm8RgAA5CBMe84KzJA2aiMRASglgFWbAl09YWQbgArmZCYjmyNg8d+G56BACYBOOLGLIkxsEbEwTAsQQTwR6dgfAG5ylTjYCwSQWBcB4GxcC5QqkkUXRifBOFHSDndPsV3AObJlkfG+biaxbYilsCgMguwoZqiwAqGgTAoTwGECgZ3D2PBbk11uQAbmYrOXYC5lAsuCaYjAvyGzjmAD7Ql+AiAcB0JYBVGy8UEsxQQKldBTUWOYtiMceI1VfI1XiUMhA6TAHkJahF+L9XKAsUQZ1bKXBapZPZUMO9CC6o2V3CUT46SzLFnPPesatmEDnnwHewB-UWLnmyLNSbNmZuzea3N+bC1puTWyVNcy4y2tNWgYNrq67ZgjDgdAYbxzupjWLf1eQPLeRIAmH2ZbLV5HJkspwbBzYMDbFO54sZVXcHMiGZQLocAZFLDoMACIQAjtQBYxtqBm2hNxF0MhxYe16vHYQW5zqF1rPQLcg9faQSNiILczgkh27AhfT7OFTYNSsmjbM-toGq3yHfVa+56BjV0H-ZB8Dtz6gAFuJQPEQ5sixr65ljqJUQXtWzQUwaNZYLDxHb23JUH69D+DOAUbFjhn2zHD3HvJSEeRbb8FOs+SGjAprDnHJQJ0si5FzlEMfFcsRkKfZvOULEFgqBgSjgSFMvEHyVmuo3OgKgFtWyEG-b+vELkS4YqJdi+duK-WWo3DShgIEvYMB0CeG2ME1HwTZi2CaUNkK3D8AOGlnDDkMr0aep9HLdkMtLHynosBqgdDpUczpJzRNnIucOGa1zXX3MIDSuTqBJ1TgAu2ztfy2AbkRR3FFwHLbcY7YQXkCQWAW23d+ezdtvyadZdpqme53qQkQuBTSFtUByIqUIPL5XGnKAIlSRUET3klZ412jcTWjAWxvPceiyg0jxqEBAYL9LuWMuBMyvjrrItcp5TF9g-KADssAihJeE8iNL5ETHvkwJ+aTs22HUh-IWchzEiuXvIACjLFt8xwbifG5m86SPjk6xo8r7nMS-fU5mI0NS0LD3bKZ0Rf3KTsPhMxHegBOQkAPB-bE8VfjooCybkJiv1bK9qyz-iwcbiM0wlJKCZOuonKSAAUioPydpVBA-MFXVcSMwddBCLQBuHjzDoBglmST1UsACU3XQFg5BaAYRZljdUiVSG6bbI+cQePrzkDBDnKGqRMDY-KetyIExCAi7FwCDI5Ajs9JO2Fi7Z6XBXbsSd27bBQz5DaLURLXTktoFS6RA+GhQGcYx7JzpFEwRJGOjNbr6rg8YBJXV+IPGQFgNV1U5gBmRWV9kHiWctw5FKHApo8BSOBtiPIAXQz4EW+7HziyJx4EjRMkHwr1sgDRxzvbOYM2Bu3OiCq8ijEYI2s98LqZwg8+wSD3U8A7AiSu7CgytEbOtxUjpEpEkdv7uolIXBRXEukKi6KDkLoqR+jXUVdn5OM4HGYqtkJY9YPANimAJueud4pSFshO4ikiAcMinA42CiugyizEB4XWCBwISBKB2ETCSiSYV+7ujAQg1wMBmYgBeI18kKIsnSC8y8Ji0QqaIWAeX+4WsCoeey4eaUketQ92L2KWImSWoW7BQeT63BUAwqoqvGWmReeIUqd6XQ3+8h2kHBuI8qzEVBg4pWSy4hsCgm3S88zKjBnAzBr2pyh8x8+QZ8GcYwvwWW7ySqTyi4K4a4JSW4LyaAdoVwF4qY6YF+XcXYaQymnE1S++r+cBpIM4jyKqS4Mu64m4+AVcucEUW0akkcqskM6ssMaUCc083gsRzyjcxccBEi1kJS68v0GCR4SRZKZ00g+U-Y34aQm6QCEQx8Y+lweAzcjAaCr05SjAEqeIkKaOLSH0UsQcssDIQUAMqRwMEcu0WRB0scxk8cOssu+sqc9kr4Qc341kuWo8mQbYnCmBkIm+uWaQAgcoSi14reYRzmygqR8ENBny6mSYTk+S4izhcRbhggHhMgBOXx5RIklRwiVMEmm6X4+Yku-4GWQEY44KNBAuDSzSNCdBSWCRbsNEHKc4vxCRaAMaJAg8jg8hJAhGGSxJrq5J0w6hLgw6GSdqh6ghaAFO1OZxtRYOi2yg-Rsg0QGANWg+WCZs5k-iJJkKjiSyFwnAA4pBzsfYLMRCJwOMWYBYxqRopKmOR+nY3YoRtKLEwhqAB8DsR8hAtQthmcGeAuQJZuFUmiBuIOtwiE34ZB7x6uUJygR+NpkIiU+B7Y9peYFMMYEwI+ICri+AESTpA4TOdJxehmXx1kMKO+5AcgAZ3J4i4p2m9EDwL0O+DM2O0QGuzkpRwJFw5kG4-RGJ8eBJqARJIA1kXQZZyQcAEGcyJA-RMyLZpIxJnypJT6JANIaATJTJQmWeYIpIOANA1AjhzMBcKRCsG0kU20Ks4M+0fUqxWs8MmUBRk5qA05VgrY1gBcY0yJ8hRc22zuTWW4s6r46oeKCQ6Ef+mQ7McBc2JO4E3p-OOg8Cgg28QkqRNs149UiKzy85YcS5GRSxq50cG56x25usxB9C1gJkC5fANs2UwFmiOwcYnCSeRp5EJpx8JQthFA+5VS-8KJV5m6SCcBk5AkiZ44b51IzEWQsQc4As32wY8E5Sz59C5Fbp0QMYnAduYirpqoSkM+nW18m+mY5glAnpeI-EKZ1FMgtFEBygDFUIxOREzExwj4ZwFw6itwclClDAVFs6UR5yQlaKM0pBgiMl4lAQggYF60aRysmR0F2RMcyUcceRGxLEclJ0YECAZBR5fAhUSEusfA+Kk5RCII1g5lngN0kMO4AA6koF8VjIuCBCkpcmwEoBAOma-ltneJvicA5kImkjNveOpeNNiEptCoxf9oqFWQAISkjLYdqnSwlVxMAQCoBuHRB0U6BdBCDoA6QKDeWqR7RyQICV4JCOBjwYAAACXYGQbIGg+oVoNQRQOoPQtgvZkKYMEMKxPlnAbIDsNERxCAAA4l5SKvrlDKFGXl1XLnwHQHGMYUvCvGvBvFvJYKmmgPuiAODGwHACAF0HWGFBqFqA6rprmOBF6j7OYC5jQEQD0HwFAHKpkhYsDQ6h+HAKALuXgHaOlEaEpHAPEIkOePWYZSYnwCCF9EyIIJTQkEkAmCABGFwIcFOazdTRzbZHgODSrrsNKDgCZI+PJToCABYhYkAA)

[Sugestão 2](https://play.codehike.org/#N4IgtgJgHiBcIFokB0B2AXAlugNgU1gAJkQBZAQwGtNUBzQ8wgYwFcBndAezGe4AdOqPBkJ4I2cgCN8hGoXQALPIQDK2ZQFFxXAE4k0bHC1pESYKjVoJyCVh263+g4egRiJ0vAhoJFXtupu2px6IGgKmBAQwkQAZuQ4bHhoTDp45OhiAILopiAATAAM+QAsCIUAbAgAjADMACr5tbC1+bCFAKwAdAAcHbUAWvqoLHwQGdm5xAXFZZU1DU0tbZ29-UNhQlBMeDp8UyQAMuk6qIQKnADu8pwMODhXvGACQiJ2XGCYAF4ZmIKE4jSTFwAE9ZGc-Kp1IQtNgQl1hkgEGg0PVbgkHtcmE5XuhmOwPt9fv85ABtNSZGHBHQAXQAFAp0Og+GxYAB6NlKHB8LoAN0yUC62LAbOEbPQLF0mASbICmSCcJ0CE4vN2vMweEuSHyAE4dQARSR4SQ6sAAMSYXwAqvUQQA5ACKOqgAAU8NUAPwAMklYAA+mxOCwdDsALzkSWcYV8fCZACUABpCCCg4QhGIbgC8LEaMpIWwmEpzIRYiFRORC6JtFIZHwdJw+Lt0GDyKgIIQ0rFdsIdvIlEksznUNg-qg2OC+8oAAY0TI6eI7NhdABWgdQU5LmHwCNQaAAkhCIuPaCxIngkymWIRLlucIR8ORTudHlxeKgcye0pPCAWi4xokOI6COOciXjoTwvC4bBJhiVyWIQ7C7OOr7vNw3x5kWAKYECoITpCFKaNSO5oAAxCRhAAELZiEyiXoQRq0DQKKoAgMJjsGtFBuB9hfgo5CqgwhDkroyj1EWeD0oyzKshy0Sqg8jY6Eu-J4IKwpshAUZsGyJ5ntpKlQN4nAIJprBgC4xIsTxXh+OZcZZvJDYZjeihCQAavUGgABqEHuADyklMiy7IaXgjmKcpApCtwGlaTpp7RPpApGSZUYsOZGCWQglx8a4mBsAgBlGfZrbtsJNGEGaOjkOZlwhJQgXSSFcl4ApSF8lF6mmdpumJbKIkILE1W1fViYMEw2I6OIdCZpC5WdvWGCNcFslha1TlKR1qnRSK3XxXpOl4EylgIBwj6ZBACC1PZvV4DurEAMJKEwlCyLE36khRp44NESnLTJoXhe1Bk7bFTA9QleBJapKWmelFlYIICCSN9v1sCVX51htOBgjQZ33BmoFcaoImEGJeDmV0hAAJqpmABLnPxygZPe6QcN+pb3HBM0o1uaNvh+HEQLAaCEGLrGktZ9Go7s-3NWtbWbSDXVxbd0OGX8qVmQjo6nQNvM-bscZEBoqA1lD34tQpGV4pw70qKTVU1Zq9U-iJQ2COgS6i+LQlpBWeIG79curUDSudTFe1q2yRWa3DNtZf7wLIzLOjG4QVpJO2r5Ww2DB8Cy17YAoQkAErpMCIdskn6BdHJbL2TlwjJsTWPLngwIdngACOp5pOOqGfF8GYexgwjtoGRiI2OVPMWRJN4Hw9Fghwi-MQAPA9AASXQqKknD3CC2LTbQAB8c-zw7i+ENUhCPf7WAzZCOb4OOpbgWB+L2Dw0bOBgzFmjLJ-aQUZXqvgrDsfYCEkjgUHkSae0FRBQAEDoPEkJy4Bwgn-PE-cXgBE8CWMskIQEvSpmJfKWDcQ-guCwH64ImBGGiIJX8FNGCKBZrBS4453CP3oICDuuE5D4WhLCXQxFUDVCpg9B+zM0yakIYbcCQilCECnNXCu6ANz1k4HifhwIQggipoArm1xIR8AyHOM4U4a5sl-ribSwAHo4hcHaZ2ABfNkNBoiCi9lAKcRiyyqRqjGc8hDwKMDsS4NMzt2xTgegSbgTjnjYKnEmZ+B9fpux0CwYEHFZDjnIG-A+mJWQ+2IKgfIVM-KNkPNOLx21fEbhfszNsg5cwt2DF-D4lComFLwiozmmJ4KrxZOIqc4z0BQHQGgGuZTADEpIQQAvBuACGdnpGA2BlLFoQBZKzCDxO-kkyC-8zhbNOds8E3iui+LQOMqczEkQX3IlfJe+Q76EH1NmGg8FISRIwAAchAmPecFZkgnNRCouJCSwCHOwS6esLINwAVzMhFRzZGxhO-L89AgBMAnHFjFkSY2CNiYJgWIIJ4JTiwLgPAG5SqqLYCwSQVL8C0pRcoVSxKLpZPgvi72u51wTLYFALuAc2T7I+DC3Eni2wNKFYQuwoZqiwAqGgTAyTUGEAwZ3D2PA-k1z+QAbmYrOXYC5lDisSc4jAcKGzjmAD7ZlBBuV0CNSchlTLsD4CIBwHQlhXVuOYtiMceILXQqtXiUMhA6TAHkJ60J7rHWEDcUQUNkqXA2pZPZUMp9CD2pOV3CUT46SbLFuvc++azmEHXnwU+wBHVuPXmyGtJbTnVtrQmuNDam3lsrY2ntYs4z+tdWgVN4a67ZgjDgdAGbxyRrzWLR1eQPLeRIAmH2HbqV5HJnspwbBzYMFaYc2M5ruDmRDMoF0OAMjvzAAiEAa7UBuOHagUdyTcRdBYcWOdDq41ED+aGndb6XB-IfQukEjY-2cEkO3YEIGfb4qbBqVkubNmLpQxWsDEHCAAvQL6ugcGMOxs3dh+oABbiUDwCOnLcaBrZG6vXocrWip1OG8O0Co2ctDfyVCMvQOR2hnAONixoz7ETj7n0cpCLoidtCQ1QrTRgV1tz7koH5WRJ5mQl61DeR8hhj5vkqKxT7cFyhYgsFQMCUcCQ1l4khQc8NG50BUAtq2QgUGYN4hciXSlcbaWtKnPRmlhBeUMBAl7BgOgTw2xgiY+CbMWwTShshW4fgBy8rGYK4VNjX1HPQNKy5vj5XsFDD0WA1QOg3PGSp5EanL6acICUHTnzhxP0M+GgFwX4UbLBagbdU4AKTunV1jcRKO6kqQ5bGTU7CC8gSCwC279vwhbtt+OzEqHNUz3O9SEiFwKaQtqgHRbShCddtf05QBEqSKjSRCgbsmZ0blm0YC2N57j0WUGkQtQgIAZcpXK7L8nw35dlcK0sCrqgAHZYBFEq3c-lDzasaevh0N5Tj3yYE-AZi7IjqQ-kLKw5ifXP3kERc1i2+ZcM5MLS07O4GLYrbMV1mLQzWvKCxZmI0XS0LD3bF55R2PKSiPhMxU+gBOQkAPB-bEGVfjokik7kJ+tTaG7avzsTicbncwIop4EfnhonKSAAUioPydpVD4-MFXVcSNiddBCLQBuYTzDoBglmPT1UsACSvXQFg5BaAYRZljdUiVmFObbI+cQPPrzkDBDnKGqRMCc7pU9yIExCBG5NwCDI5BfuFYB-ZoDGBgc+LlWD4r+Q2i1AqwKuHaAEekTq9fCobyNDIKk1jmzzEKIryOrw1ROWUmhcm-EWTSCUHO46cwVzaqx+yDxLOW4OilDgXMag8cDOVHkALm58Ci-dj51GZVMsRomR77t62eBo4D3tnMGbP30XRCjZJRiMEi3N+Fy84QG-YJB42cQdgfJXcwoGU0Q2ctwqQ6QlISQK+qenM-Mi2FcJc7OfOViNiWKbAquk4ZwkmGqtkJY9YPAXimAQePud4zSgKq26iAcWinAR2eiughizEB4FBNc1BtB2EAiBiSY4BqejAQg1wzSmY2BeIn8WKIs-Km8O8Ti0Q5atyueGiYqgOBeeW9SxeoOaUbAoYtQEOsO1WzEsh-28h-e761yw46qcm+euWeB3A2GXQti4a2kRhwGEmreGqd202jhimOh4hYqkhnA0huhiOC8S8EObyVoYwvwrOqiJqwKi4K4a4TSW4oKaAdoVwF4qY6YoBXcXYaQFmnEnSP+7O-OQkM4QKZqS4Vu64m4+AVcucEUW0akkcqskM6ssMaUCc083gpRIKjcxcRRai1kTSGSuwZCR4VR7KZ00g+U-Y34aQV6CCEQS8x+lweAzcjAJCr0dKjAWqeIWKzO3M9Ac0UsQcssDIQUAMtRwMEcu0TRB0scxk8cOs1u+sqc9kr4Qc341kHWo8mQbY4iTBkIb+HWaQAgcoBi14S+eREWygtR8EIhUKNmSYTktSURXRsRFRCRMgyBFBAxCiv0VMumV6X4+Y5u-4zWQEY4GKIheufSgyVwpS-KtyFRbsNE0qc4ZRcRggaAeaJAg8jgShJARA86YsPJ4aAp0wHh0y96ZSAaj6XhaAYuku-xoxxON2yg6xsg0QGA42e+FCZs5ksSvJWK-ieyFwnAA4vBzsfYLMDCJwOMWYBYvqRobKNms+WR3YuRfKteqm9eSOS8PQoRAQkRSBvRZwjAUs5ifuhOtwiE34fBCJruhJyg-+WJkIiU7B7YEZeYFMMYEwAS4EQSzw+AaS0ZA4CuEpGuyJ1kuKn+5AcgmZqpqihpDm9EDwL0n+DMnO0QbuzkIZq2Fw5kG46xYhsOFRXJKIIA1kXQ-ZyQcAjGwpIA6xGys5pIPJUKfJuWJANIaAMpMpym-KXeQkOANA1AkRb+NRCsG0kU20Ks4M+0fUdxWs8MmUHRR5qAJ5VgrY1gBcY0VJShRcb2ies2W4+6r46oDKCQ6EV+8gUMc+yJl2Qu4EKZbWOguCggJ8QktRNs149URKIK55YcV5DR1xt50cD5Dxz5us3BvC1gJkF5fANs2UOF5iOwcY4ideqA6mQRhAOobyFA75HSsC1JQFV6BCRRR5AkVZ448F1IzEWQsQc4AsGOwY8EdKmQHAsJxMuxWYMYnAMeKicZqoSkl+K2n8b+mY5glASZeI-EtZIlMgYlRBygklUIguREzExwj4ZwFwpitwFlVlDAwl+6RRpIHyOl5KM0vB8iZlhlAQgg+F60dRysjRJFzRMcyUccbRjxLEFlJ0YECAfBX5fAhUSEusfAjKR5DCII1ggVngN0kMO4AA6koMiVjIuCBG-F8mwEoBAA2cGW9m-icIPrSVwudveI5eNNiOZjilJTjoqMOagAAISkhuGuDE5VxMAQCoBxHRDiU6BdBCB5a0AKBpWqR7RyQIBj4JCOBjwYAAACXYGQbIGg+oVoNQRQOoPQtga5WKYMEMtx6VnAbIDsNE3xCAAA4qlWqr7lDKFMPlOqdCSV0HwHQHGBvD4bvPvIfMfJYOWmgPeiAODGwHACAEjWkOqJqNdU5rmOBDGj7OYJFjQEQD0HwFAAauUm4vjUGh+HAKAK+XgHaOlEaEpHAPEIkOeBOd5U4nwCCF9EyIICLQkEkAmCABGFwIcMeQrWLcrbZHgMTU7rsNKDgCZI+JZToCAG4m4kAA)

File: docs/guides/vtex-io/Storefront-Guides/concepts-1/vtex-io-documentation-making-a-custom-component-editable-in-site-editor.mdx [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+1. Create a new folder in the `/react` root directory. Follow the `react/components/{ComponentName}/index.tsx` pattern. For example, for a component named `CustomComponent`, the folder structure is as follows:
+   
+2. Open the `index.tsx` file and define your custom component.
+
+```txt
+  react
+     ┣ 📂 components
+        ┣ 📂 CustomComponent
+           ┣ index.tsx
+```
```
[the remark this answers is not in the document]
Comment (barbara-celi) [reply docs/guides/vtex-io/Storefront-Guides/concepts-1/vtex-io-documentation-making-a-custom-component-editable-in-site-editor.mdx]: Pra manter a continuidade do bloco, fiz de outra forma. De fato estava um pouco confuso por causa da estrutura da frase, então mudei para `The folder structure for a component named CustomComponent is shown alongside`.

**Review by Mariana-Caetano — APPROVED** [review]

File: docs/guides/vtex-io/Storefront-Guides/concepts-1/vtex-io-documentation-making-a-custom-component-available-in-site-editor.mdx [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+1. Create a new folder in the `/react` root directory. Follow the `react/components/{ComponentName}/index.tsx` pattern. See alongside a folder structure example for a component named `CustomComponent`.
+   
+2. Open the `index.tsx` file and define your custom component.
+
+```txt
+  react
+     ┣ 📂 components
+        ┣ 📂 CustomComponent
+           ┣ index.tsx
```
Comment (henriquessb) [inline docs/guides/vtex-io/Storefront-Guides/concepts-1/vtex-io-documentation-making-a-custom-component-available-in-site-editor.mdx:39]:
```suggestion
           ┗ 📄 index.tsx
```

File: docs/guides/vtex-io/Storefront-Guides/concepts-1/vtex-io-documentation-making-a-custom-component-available-in-site-editor.mdx [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+Now, you need to reference your custom component in the [`interfaces.json` file](https://developers.vtex.com/docs/guides/vtex-io-documentation-interface) within the `/store` folder. This file establishes the relationship between a block and a React component, allowing the [store builder](https://developers.vtex.com/docs/guides/vtex-io-documentation-store-builder) to build the store's frontend.
+
+In the app repository where you are developing your custom component, open the `interfaces.json` file within the `/store` folder. Declare the schema definitions for your component as follows:
+
+```json store/interfaces.json
+{
+"custom-component": {
+   "component": "CustomComponent",
+   }
```
Comment (henriquessb) [inline docs/guides/vtex-io/Storefront-Guides/concepts-1/vtex-io-documentation-making-a-custom-component-available-in-site-editor.mdx:151]: Corrigindo indentação

```suggestion
  "custom-component": {
    "component": "CustomComponent",
  }
```

File: docs/guides/vtex-io/Storefront-Guides/concepts-1/vtex-io-documentation-making-a-custom-component-available-in-site-editor.mdx [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+
+### Step 8 - Using the component in a store page
+
+To use the new component, declare it in the desired page template. For example, to use the `CustomComponent` on the store’s main page, the `custom-component` block must be declared within the `home` block:
+
+```json
+{
+"store.home": {
+   "blocks": ["custom-component"]
+}
```
Comment (henriquessb) [inline docs/guides/vtex-io/Storefront-Guides/concepts-1/vtex-io-documentation-making-a-custom-component-available-in-site-editor.mdx:167]: Corrigindo indentação

```suggestion
  "store.home": {
    "blocks": ["custom-component"]
  }
```

File: docs/guides/vtex-io/Storefront-Guides/concepts-1/vtex-io-documentation-making-a-custom-component-available-in-site-editor.mdx [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+
+By [linking the app](https://developers.vtex.com/docs/guides/vtex-io-documentation-linking-an-app), your component will be available for visualization and test in Site Editor within the corresponding [development workspace](https://developers.vtex.com/docs/guides/vtex-io-documentation-creating-a-development-workspace).
+
+---
+
+### Step 9 - Making your component available in the live store’s Site Editor
+
+After configuring and testing your component, deploy the new version of your app to make it available in the live store’s Site Editor.
+
+Learn how to make it available in the [Deploying a new app version](https://developers.vtex.com/docs/guides/vtex-io-documentation-making-your-new-app-version-publicly-available) guide.
```
Comment (henriquessb) [inline docs/guides/vtex-io/Storefront-Guides/concepts-1/vtex-io-documentation-making-a-custom-component-available-in-site-editor.mdx:179]: Adicionando quebra de linha

```suggestion
Learn how to make it available in the [Deploying a new app version](https://developers.vtex.com/docs/guides/vtex-io-documentation-making-your-new-app-version-publicly-available) guide.

```

**Review by henriquessb — APPROVED** [review]

**Pull request merged**